sdk/core/System.BinaryData/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("System.BinaryData.Tests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d15ddcb29688295338af4b7686603fe614abd555e09efba8fb88ee09e1f7b1ccaeed2e8f823fa9eef3fdd60217fc012ea67d2479751a0b8c087a4185541b851bd8b16f8d91b840e51b1cb0ba6fe647997e57429265e85ef62d565db50a69ae1647d54d7bd855e4db3d8a91510e5bcbd0edfbbecaa20a7bd9ae74593daa7b11b4")]
```

Comment:
InternalsVisibleTo isn't permitted once it moves to dotnet/runtime, please don't use it here.

Reply:
This is needed for the ReadOnlyMemoryStream tests.

Reply:
> This is needed for the ReadOnlyMemoryStream tests.

Why?  It doesn't have surface area different from a stream, other than the ctor.  It can be tested indirectly using the ToStream() method (which white-box testing says `new BinaryData(buf).ToStream()` is the same as `new ReadOnlyMemoryStream(buf)`) or more explicitly with reflection.

Assuming that ReadOnlyMemoryStream sticks around, changing to one of those two models will be required for the transition to dotnet/runtime.

Reply:
Ok, will remove.